packages/api/src/controllers/helpers.test.ts

```diff
@@ -182,6 +183,40 @@ describe("controllers/helpers", () => {
       );
     });
   });
+
+  describe("getObjectStoreS3Config", () => {
+    it("should handle a few URL formats", async () => {
+      const result = await getObjectStoreS3Config({
```

Comment:
No idea why that function was made async lol. Can probably make it sync if you can

Reply:
Haha yeah I reached the same conclusion